app-cryptography/signing-party/autobuild/build

```diff
@@ -0,0 +1,42 @@
+cd "$SRCDIR"
+
+abinfo "Running make to generate files..."
+make
+
+abinfo "Installing files..."
+mkdir -pv "$PKGDIR"/usr/bin
+mkdir -pv "$PKGDIR"/usr/share/man/man1
+
+# To make the following for loop work.
+install -Dvm755 gpgwrap/bin/gpgwrap gpgwrap/gpgwrap
+
+for binname in \
+	caff gpg-key2latex gpg-key2ps gpg-mailkeys gpgdir gpglist \
+	gpgparticipants gpgsigs gpgwrap keyanalyze keyart keylookup \
+	sig2dot springgraph ; do
+	install -Dvm755 "$SRCDIR"/$binname/$binname "$PKGDIR"/usr/bin/$binname
+	find $binname -type f -iname '*.1' -exec cp -v {} "$PKGDIR"/usr/share/man/man1/ \;
+done
+
+abinfo "Installing extra files..."
+for binpath in \
+	caff/pgp-clean caff/pgp-fixkey gpgparticipants/gpgparticipants-filter gpgparticipants/gpgparticipants-prefill \
+	keyanalyze/pgpring/pgpring keyanalyze/process_keys ; do
+	install -Dvm755 $binpath /usr/bin/${binpath/#*\//}
+done
+
+# List obtained from Arch Linux:
+# https://gitlab.archlinux.org/archlinux/packaging/packages/signing-party/-/blob/main/PKGBUILD?ref_type=heads
+abinfo "Installing examples..."
+mkdir -pv "$PKGDIR"/usr/share/doc/$PKGNAME/{caff,gpgsigs,keyanalyze,keyart}
+for example in \
+	caff/caffrc.sample gpgsigs/gpgsigs-lt2k5.txt gpgsigs/gpgsigs-lt2k5-annotated.txt keyanalyze/analyze.sh \
+	keyanalyze/allkeys.sh keyanalyze/scripts keyanalyze/willy keyart/doc ; do
+	cp -rv $example "$PKGDIR"/usr/share/doc/$PKGNAME/$example
+done
+# Remove keyart.1 in examples
```

Comment:
Why?

Reply:
It is a manual page, will fix it later.

Reply:
> It is a manual page, will fix it later.

Just be sure that you know why it was being removed.